### PR TITLE
[stable21] Bump guzzlehttp/guzzle and guzzlehttp/psr7

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,10 @@
 		"platform": {
 			"php": "7.3.0"
 		},
-		"sort-packages": true
+		"sort-packages": true,
+		"allow-plugins": {
+			"cweagans/composer-patches": true
+		}
```
Comment (come-nc) [inline composer.json:16]: Why this change? I see no patches added in the PR

**Comment by come-nc** [comment]
CI fails on test classes which were ignored and should not, or the other way around.

File: composer.json (hunk 1 above)
Comment (nickvergessen) [reply composer.json:16]: It is there in stable22 and also the composer.patches.json file is here on stable21 so every time the composer is executed you will be asked to trust it:
https://github.com/nextcloud/3rdparty/blob/stable21/composer.patches.json

But can also remove it.

**Comment by nickvergessen** [comment]
Fixed by backporting: https://github.com/nextcloud/3rdparty/pull/905

**Review by vitormattos — APPROVED** [review]

**Review by come-nc — APPROVED** [review]
